### PR TITLE
using_images/db_images/mysql.adoc: describe new parameters

### PR DESCRIPTION
**Pull request by php-coder**
Documents 8 new parameters that were added to MySQL image in https://github.com/openshift/mysql/pull/130

### CODE REVIEW

**Comment by php-coder** [comment]
PTAL @openshift/team-documentation

**Comment by php-coder** [comment]
Ping @openshift/team-documentation

**Comment by mfojtik** [comment]
LGTM @adellape merge pls.

File: using_images/db_images/mysql.adoc [hunk 1]
```diff
@@ -175,8 +179,41 @@ MySQL settings can be configured with the following environment variables:
...
+|The size of each log file in a log group.
+|8M (or 15% of available memory)
+
+|`*MYSQL_INNODB_LOG_BUFFER_SIZE*`
+|The size of the buffer that InnoDB uses to write to the log files on disk.
+|8M (or 15% of available memory)
 |===
 
+Some of the memory-related parameters have two default values. The fixed value
+is used when container does not have link:../../dev_guide/compute_resources.html#memory-limits[memory limits]
```
Comment (adellape) [inline using_images/db_images/mysql.adoc]: s/container/a container/

File: using_images/db_images/mysql.adoc [hunk 2]
```diff
@@ -175,8 +179,41 @@ MySQL settings can be configured with the following environment variables:
...
+assigned. The other value is calculated dynamically during container's startup
```
Comment (adellape) [inline using_images/db_images/mysql.adoc]: s/container's/a container's/

**Comment by adellape** [comment]
@php-coder @mfojtik Minor nits, otherwise LGTM.

Any idea on a target release version for this for OSE? I'll milestone it generally "Future Release" for now and keep an eye out.

File: using_images/db_images/mysql.adoc (hunk 1 above)
Comment (php-coder) [inline using_images/db_images/mysql.adoc]: Fixed, thanks!

**Comment by php-coder** [comment]
@adellape The related Trello card (https://trello.com/c/JH769zUA/893-3-update-mysql-image-to-autoconfigure-based-on-available-memory-evg) has `targeted-3.3` label. Is it what you're asking for?

**Comment by adellape** [comment]
@php-coder Yep, thanks! Also there was one other minor nit inline (two total).

File: using_images/db_images/mysql.adoc (hunk 2 above)
Comment (php-coder) [inline using_images/db_images/mysql.adoc]: Fixed, thanks!

**Comment by adellape** [comment]
[rev_history]
|link:../using_images/db_images/mysql.html[MySQL]
|Updated the Additional MySQL Settings table in the link:../using_images/db_images/mysql.html#environment-variables[Environment Variables] section with new variables.
%

**Pull request merged**